app/views/home/index.html.slim

```diff
@@ -1,110 +1,101 @@
-#myCarousel.carousel.slide.px-1[data-bs-ride="carousel"]
-  .carousel-indicators
-    button.active[type="button" data-bs-target="#myCarousel" data-bs-slide-to="0" aria-current="true" aria-label="Slide 1"]
-    button[type="button" data-bs-target="#myCarousel" data-bs-slide-to="1" aria-label="Slide 2"]
-    button[type="button" data-bs-target="#myCarousel" data-bs-slide-to="2" aria-label="Slide 3"]
-  .carousel-inner
-    .carousel-item.active
-      = image_tag('banner1.jpg', class: 'd-block w-100', alt: t('.banner1.alt_text'))
-      .carousel-caption.d-none.d-md-block
-        .background-transparent
-          h1 = t('.banner1.header')
-          p = t('.banner1.paragraph')
-      .copyright
-        | &copy;
-        = " "
-        = "Bengdt Nyman"
-        = ' / '
-        = link_to "cc", 'https://creativecommons.org/licenses/by/2.0/'
-    .carousel-item
-      = image_tag('banner2.jpg', class: 'd-block w-100', alt: t('.banner2.alt_text'))
-      .carousel-caption.d-none.d-md-block
-        .background-transparent
-          h1 = t('.banner2.header')
-          p = t('.banner2.paragraph')
-      .copyright
-        | &copy;
-        = " "
-        = "Solent Creatives"
-        = ' / '
-        = link_to "cc", 'https://creativecommons.org/licenses/by/2.0/'
-    .carousel-item
-      = image_tag('banner3.jpg', class: 'd-block w-100', alt: t('.banner3.alt_text'))
-      .carousel-caption.d-none.d-md-block
-        .background-transparent
-          h1 = t('.banner3.header')
-          p = t('.banner3.paragraph')
-      .copyright
-        | &copy;
-        = " "
-        = "Tim Peacock"
-        = ' / '
-        = link_to "cc", 'https://creativecommons.org/licenses/by/2.0/'
-  button.carousel-control-prev[type="button" data-bs-target="#myCarousel" data-bs-slide="prev"]
-    span.carousel-control-prev-icon[aria-hidden="true"]
-    span.visually-hidden
-      = t('common.button.previous')
-  button.carousel-control-next[type="button" data-bs-target="#myCarousel" data-bs-slide="next"]
-    span.carousel-control-next-icon[aria-hidden="true"]
-    span.visually-hidden
-      = t('common.button.next')
+section#hero.hero
+  .container.mt-3
+    .row
+      .col-lg-6.col-md-7.col-sm-12.my-auto
+        .container-fluid.px-4.py-5.my-5.text-center
+          .lc-block.mb-4
+            h2.display-3.fw-bold
+              i.fa.fa-solid.fa-anchor.me-1
+              = t('.hero.title1')
+              span.text-secondary
+                = t('.hero.title2')
+          .lc-block.mx-auto.mb-5
+            p.lead
+              = t('.hero.subtitle')
+          - unless user_signed_in?
+            .lc-block.d-grid.gap-2.d-sm-flex.justify-content-sm-center.mb-5
+              = link_to 'Sign Up', new_user_registration_path, class: 'btn btn-main btn-light btn-lg px-4 gap-3'
```

Comment:
Here is another translation that should be moved (or, better, reused from the locales). Please check the remaining code once again.